src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -126,6 +149,86 @@ public void DrawGeometry(IBrush brush, IPen pen, IGeometryImpl geometry)
                 case Line myLine:
                     DrawLineInternal(pen, myLine);
                     break;
+                case StreamGeometryImpl streamGeometry:
+                {
+                    pen = pen ?? new Pen(brush);
+
+                    var extractColorCheckPlatformSupported =
+                        ExtractColorOrNullWithPlatformCheck(pen, out var lineStyle);
+                    if (extractColorCheckPlatformSupported == null)
```

Comment:
Does it make sense to ignore no supported? Could be better to notify it's not going to be painted with an exception?